pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -539,11 +537,22 @@ func buildAntiAffinity(labelMap map[string]string) *core_v1.PodAntiAffinity {
 	}
 }
 
-func buildPodDisruptionBudgetSpec(labelMap map[string]string) policy_v1.PodDisruptionBudgetSpec {
+func buildPodDisruptionBudgetSpec(labelMap map[string]string, minReplicas int32) policy_v1.PodDisruptionBudgetSpec {
+	// Make sure we can drain nodes if the minReplicas is < 3
+	var pdbPercentage string
+	switch minReplicas {
+	case 0, 1:
+		pdbPercentage = "100%"
```

Comment:
should be 0 here since it's "MinAvailable" ?

Reply:
Whoops, yes percentages were inverted.

Reply:
Fixed now.